cli/test_pack_python3.bats

```diff
@@ -66,7 +62,7 @@ skip_tests_if_python3_is_not_available_or_if_already_running_under_python3() {
 @test "packs.setup_virtualenv with python3 flag works" {
 	skip_tests_if_python3_is_not_available_or_if_already_running_under_python3
 
-	SETUP_VENV_RESULTS=$(st2 run packs.setup_virtualenv packs=examples python3=true -j)
+	SETUP_VENV_RESULTS=$(run_command_and_log_output st2 run packs.setup_virtualenv packs=examples python3=true -j)
 	run eval "echo '$SETUP_VENV_RESULTS' | jq -r '.result.result'"
 	assert_success
```

Comment:
In this specific case, if we really need `jq`, this construct with `run | jq + assert` should be replaced in favor of creating custom `assert_output_jq` function that will help us to avoid another `run eval` and override original test results/outputs.

Reply:
I just read this comment after coming to the same conclusion. :+1: